### ISSUE TITLE
Fix assignment on editing github issue PR description

### ISSUE THREAD

**Issue by rndquu** [ubiquity-os-marketplace/command-start-stop#84]
There's [this](https://github.com/ubiquity-os/ubiquity-os-plugin-installer/issues/4) issue which was initially assigned to `Keyrxng`.

What happened:
1. I edited [this](https://github.com/ubiquity-os/ubiquity-os-plugin-installer/pull/12#issue-2642288540) PR comment
2. `ubiquity-os-beta` suddenly [assigned](https://github.com/ubiquity-os/ubiquity-os-plugin-installer/issues/4#issuecomment-2478494730) me to the issue while the expected behavior was to keep `Keyrxng` as the original assignee

P.S. Not sure what's the correct repository for this issue, https://github.com/ubiquity-os-marketplace/command-start-stop or https://github.com/ubiquity-os-marketplace/daemon-disqualifier

**Comment by 0x4007** [comment]
Looks like the logic to capture the closing keywords also works on edits. I suppose that it should check if it is currently assigned, and if so, to not clobber assignment.

**Comment by Sadaf-A** [comment]
@rndquu @0x4007 So this issue belongs to the same repo right?

**Comment by rndquu** [comment]
> @rndquu @0x4007 So this issue belongs to the same repo right?

I'm not 100% sure, https://github.com/ubiquity-os-marketplace/command-start-stop or https://github.com/ubiquity-os-marketplace/daemon-disqualifier

**Comment by 0x4007** [comment]
@gentlementlegen i am guessing it's command start stop but would be great if you can confirm.

**Comment by gentlementlegen** [comment]
Yes it is `command-start-stop`, the disqualifier never assigns users. I suppose this behavior has something to do with the flow on pull-request linking to assign users.

**Comment by gentlementlegen** [comment]
I've investigated this, and this happens because there is [logic](https://github.com/ubiquity-os-marketplace/command-start-stop/blob/b3405d795416687afbeb080092dce2c7b33111c1/src/plugin.ts#L23) associated to `pull_request.edited`. This was implemented because if you first open a PR without linking the issue, then edit your pull-request to add `Resolves #--` we need to listen for this event to properly assign a user. I suppose the fix would be to assign the author of the pull-request instead of the user who edited it.
